builder/builder-next/controller.go

```diff
@@ -570,30 +570,14 @@ func getLabels(opt Opt, labels map[string]string) map[string]string {
 	return labels
 }
 
-func getCDIManager(specDirs []string) (*cdidevices.Manager, error) {
-	if len(specDirs) == 0 {
+func getCDIManager(opt Opt) (*cdidevices.Manager, error) {
+	if opt.DisableCDI {
 		return nil, nil
 	}
-	cdiCache, err := func() (*cdi.Cache, error) {
-		cdiCache, err := cdi.NewCache(cdi.WithSpecDirs(specDirs...))
-		if err != nil {
-			return nil, err
-		}
-		if err := cdiCache.Refresh(); err != nil {
-			return nil, err
-		}
-		if errs := cdiCache.GetErrors(); len(errs) > 0 {
-			for dir, errs := range errs {
-				for _, err := range errs {
-					log.L.Warnf("CDI setup error %v: %+v", dir, err)
-				}
-			}
-		}
-		return cdiCache, nil
-	}()
-	if err != nil {
-		return nil, errors.Wrapf(err, "CDI registry initialization failure")
-	}
+
+	// Use the default CDI cache that was configured during daemon startup
+	cdiCache := cdi.GetDefaultCache()
```

Comment:
Does this mean that the builder and the daemon share the singleton CDI cache that is instantiated the first time that `GetDefaultCache` is called?